### ISSUE TITLE
GML Reader fails to parse some of the formats given on Wiki

### ISSUE THREAD

**Issue by satishbhor** [locationtech/jts#552]
```
<gml:Point gml:id="p21" srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
    <gml:coordinates>45.67, 88.56</gml:coordinates>
 </gml:Point>
```
Another example is
```
<gml:Point gml:id="p21" srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
    <gml:pos srsDimension="2">45.67 88.56</gml:pos>
 </gml:Point>
```
Reference Shapes are available here
https://en.wikipedia.org/wiki/Geography_Markup_Language

**Comment by dr-jts** [comment]
The JTS GML parser only handles GML 2.0.  `<pos>` is a GML 3.0 construct, so the second example is out of scope.

The first issue looks like it's a bug, where the `coordinates` parser is not handling spaces between ordinates.  Note that when the spaces are removed it works:
```
<gml:Point gml:id="p21" srsName="http://www.opengis.net/def/crs/EPSG/0/4326">
    <gml:coordinates>45.67,88.56</gml:coordinates>
 </gml:Point>
```

**Comment by satishbhor** [comment]
@dr-jts should I contribute to this issue?

**Comment by dr-jts** [comment]
That's ok, I have a fix and am commiting it now

**Comment by satishbhor** [comment]
ohh cool! Thanks!

**Comment by dr-jts** [comment]
The PR is here: https://github.com/locationtech/jts/pull/553
 Am waiting to merge it until I can figure out why the CI build is not running.

**Comment by satishbhor** [comment]
@dr-jts also does it retrieves srsName correctly?

**Comment by satishbhor** [comment]
I had observed geomtry SRID sets to zero even after above example has srsName

**Comment by dr-jts** [comment]
The parser handles `srsName` values of the form "int" or "url#int".   It doesn't handle the form `http://www.opengis.net/def/crs/EPSG/0/4326` given in the wikipedia examples.

**Comment by dr-jts** [comment]
I will improve this to use the final number in the `srsName` value.  That should handle most cases.

**Comment by satishbhor** [comment]
Thanks a lot @dr-jts ! Appreciate your prompt help!

**Comment by dr-jts** [comment]
Fixed by #553
